File: cigarros.c
#include <stdio.h>

int anosPerdidos(int cigarros, int anos){

    int cigarrosFumados = cigarros * 365 * anos;
    int minutosPerdidos = cigarrosFumados * 10;
    int diasPerdidos = minutosPerdidos / (24 * 60);

    printf("Foram perdidos: %d", diasPerdidos);
    printf(" dias e %d", minutosPerdidos);
    printf(" minutos");
}

int main(){
    int cigarro = 0;
    int anos = 0;


    printf("Quantos cigarros são fumados por dia: ");
    scanf("%d", &cigarro);

    printf("Quantos anos voce ja fumou: ");
    scanf("%d", &anos);

    anosPerdidos(cigarro, anos);
}

File: atividade_3/cigarros.py
cigarros_por_dia = int(input("Quantidade de cigarros fumados por dia: "))
anos_fumados = int(input("Quantos anos você já fumou: "))

total_cigarros = cigarros_por_dia * 365 * anos_fumados
tempo_perdido_minutos = total_cigarros * 10  

tempo_perdido_dias = tempo_perdido_minutos / (24 * 60)  


print(f"Um fumante perderá aproximadamente {tempo_perdido_dias:.2f} dias de vida.")
